app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -65,6 +65,16 @@ def promote
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_view_versions/:id", N_("Update a content view version.")
+    param :id, :number, :desc => N_("Content view version identifier"), :required => true
+    param :description, String, :desc => N_("The description for the content view version")
+    def update
+      history = Katello::ContentViewHistory.find_by(:katello_content_view_version_id => @version.id)
+      history.notes = params[:description]
+      history.save!
+      respond_for_show(:resource => @version)
```

Comment:
You'd want to lookup the correct history item (the publish).  See:  https://github.com/Katello/katello/blob/master/app/models/katello/content_view_version.rb#L118

something like  @version.history.publish.first.notes = params[:description]